storage/storage_interface.ml

```diff
@@ -99,6 +99,7 @@ end = struct
 end
 
 type sr = Sr.t
+let sr_pp : Format.formatter -> sr -> unit = fun ppf sr -> Format.fprintf ppf "%s" (Sr.string_of sr)
```

Comment:
Should the `pp` be inside the module itself? (See http://erratique.ch/software/fmt/doc/Fmt.html#nameconv)